setup.py

```diff
@@ -1,25 +1,19 @@
 import os
```

Comment:
FYI - we're keeping `setup.py` because `kf-model-fhir/kf_model_fhir` is still going to be a Python package with the FHIR service plugin in it. The only change that needs to be made is removing the CLI entrypoint